jupyterhub/tests/test_api.py

```diff
@@ -466,8 +433,8 @@ def test_never_spawn(app, io_loop, no_patience, request):
 
     db = app.db
     name = 'badger'
-    user = add_user(db, app=app, name=name)
-    r = api_request(app, 'users', name, 'server', method='post')
+    add_user(db, app=app, name=name)
+    api_request(app, 'users', name, 'server', method='post')
```

Comment:
FWIW, I find 'unused variable' linters mostly unproductive on test code, since it's common to change a test to use the result or not, and having to change the line where the result is computed based on later code seems like a waste to me.